service.subtitles.supersubtitles/service.py

```diff
@@ -0,0 +1,577 @@
+# -*- coding: utf-8 -*- 
+
+import time
+
+start = time.time()
+
+import os
+import sys
+py2 = sys.version_info.major == 2
+import shutil
+import unicodedata
+import os.path
+import re
+
+from kodi_six import xbmc, xbmcvfs, xbmcaddon, xbmcgui, xbmcplugin
+from kodi_six.utils import py2_encode, py2_decode
+
+import json
+
+import urllib
+if py2:
+    from urllib import unquote, unquote_plus, urlencode, quote_plus
+    from urllib2 import urlopen, Request, HTTPError, URLError
+else :
+    from urllib.parse import unquote, unquote_plus, urlencode, quote_plus
+    from urllib.request import urlopen, Request
+    from urllib.error import HTTPError, URLError
+
+__addon__ = xbmcaddon.Addon()
+#__author__ = __addon__.getAddonInfo('author')
```

Comment:
I would _suggest_ to not include commented code lines in a release version. You can always look at your own source control history to see what was there before.